src/maintainer/knowledge_base.rst

```diff
@@ -1527,3 +1527,32 @@ Using ``arch_rebuild.txt``
 
 You can add a feedstock to ``arch-rebuild.txt`` if it requires rebuilding with different architectures/platforms (such as ppc64le or aarch64). To add the feedstock to ``arch_rebuild.txt``, open a PR to the `conda-forge-pinning-feedstock repository <https://github.com/conda-forge/conda-forge-pinning-feedstock>`__.
 Once the PR is merged, the migration bot goes through the list of feedstocks in ``arch_rebuild.txt`` and opens a migration PR for any new feedstocks and their dependencies, enabling the aarch64/ppc64le builds.
+
+.. _migrations_and_migrators:
+
+Migrators and Migrations
+========================
+
+When any changes are made in the global pinnings of a package, then the entire stack of the packages that need that package on their ``host`` section would need to be updated and rebuilt.
+Doing it manually can be quite tedious, and that's where migrations come to help. Migrations automate the process of submitting changes to a feedstock and are an integral part of the ``regro-cf-autotick-bot``'s duties.
+
+There are several kinds of migrations, which you can read about in `Making Migrators <https://regro.github.io/cf-scripts/migrators.html>`__. To generate these migrations, you use migrators, which are bots that automatically create pull requests for the affected packages in conda-forge.
+To propose a migration in one or more pins, the migrator issues a PR into the pinning feedstock using a yaml file expressing the changes to the global pinning file in the migrations folder.
+Once the PR is squashed and merged, the migrator bot uses the yaml file to calculate which packages need to be rebuilt and starts issuing PRs with the new pinning as a local pinning file to the feedstocks.
```

Comment:
Why do we have to mention "squash" here? Please don't... It's a very bad practice against tracking changes, sometimes a migration PR needs additional work by the maintainers. I understand there are "purists" out there who would defend the cleanness of the commit history, but really it's unnecessary to advocate.
```suggestion
Once the PR is merged, the migrator bot uses the yaml file to calculate which packages need to be rebuilt and starts issuing PRs with the new pinning as a local pinning file to the feedstocks.
```

Reply:
(This sentence is also a bit weird to me, IIRC we build the whole dependency graph first, and then the bot walks through the graph to migrate all nodes one by one. @beckermr can chime in here.)

Reply:
Hi @leofang! Thank you so much for reviewing the PR. 
Rephrasing it to - "Once the PR is merged, the dependency graph is built. After that, the bot walks through the graph and migrates all the nodes (feedstocks) one by one and issues PRs for those feedstocks." would this be correct? 
@kathatherine, can you also please take a look and let me know if something needs to be changed in this?

Reply:
@ssurbhi560 Let's avoid two "and"s in the same list.

"Once the PR is merged, the dependency graph is built. After that, the bot walks through the graph, migrates all the nodes (feedstocks) one by one, and issues PRs for those feedstocks."

Would also love @beckermr's check of this.

Reply:
Yes @leofang is correct. We issue PRs in topological order in the graph and downstream packages have to wait for their upstream parents before they get a PR.